template/index.js

```diff
@@ -0,0 +1,42 @@
+import { File } from "@asyncapi/generator-react-sdk";
+import filter from  '../filters/all.js';
+import { HTML, Head, Body } from "../components/common";
+
+export default function Index({ asyncapi, params }) {
+  return (
+    <File name="index.html">
+      <HTML>
+        <Head
+          title={`${asyncapi.info().title()} ${asyncapi.info().version()} documentation`}
+          cssLinks={params.singleFile ? [filter.includeFile("template/css/global.min.css"),filter.includeFile("template/css/asyncapi.min.css")] : ["css/global.min.css","css/asyncapi.min.css"]}
```

Comment:
As in previous comment. When you have set `singleFile` you should include that css files inside `<head>` as content, not path to that files. Please fix it. Of course without `singleFile` you should create proper css links like current. You need to update `<Head>` component.